### PR TITLE
feat(css-structure): add exercise

### PR DESCRIPTION
**Pull request by aclaes**

### CODE REVIEW

**Review by djfarly — APPROVED** [review]
[LGTM](https://neuefische-students.slack.com/archives/C034DRT5W0L/p1658935386970089)

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+nav li {
+	padding: 0.5rem 1.5rem 0.5rem 0;
+}
+
+/* -------------------------------------------------------- */
+/* The exercise starts here. */
+/* -------------------------------------------------------- */
+
+/* 1. This styles should apply to the headline */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:52]:
```suggestion
/* 1. These styles should apply to the headline */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	font-size: 2rem;
+	margin: 0 2rem 0 0;
+	color: #2673bb;
+}
+
+/* 2. This styles should apply to all links */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:60]:
```suggestion
/* 2. These styles should apply to all links */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #ff00e6;
+}
+
+/* 3. This styles should apply to all external links (Use can use the attribute selector) */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:66]:
```suggestion
/* 3. These styles should apply to all external links (you can use the attribute selector) */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #ff8000;
+}
+
+/* 4. This styles should apply to all links in the main navigation */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:72]:
```suggestion
/* 4. These styles should apply to all links in the main navigation */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #ffffff;
+	text-decoration: none;
+}
+
+/* 5. This styles should apply to the main navigation link corresponding to the current page */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:79]:
```suggestion
/* 5. These styles should apply to the main navigation link corresponding to the current page */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #22e781;
+}
+
+/* 6. This styles should apply to all links in the footer */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:85]:
```suggestion
/* 6. These styles should apply to all links in the footer */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: inherit;
+}
+
+/* 7. This styles should apply to all paragraphs and all lists (Please use a class selector) */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:91]:
```suggestion
/* 7. These styles should apply to all paragraphs and all lists (please use a class selector) */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #2c3f3f;
+}
+
+/* 8. This styles should apply to all "notes" */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:97]:
```suggestion
/* 8. These styles should apply to all "notes" */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	font-size: 0.8em;
+	color: #808080;
+}
+
+/* 9. This styles should apply to the welcome text */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:104]:
```suggestion
/* 9. These styles should apply to the welcome text */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	font-size: 1.2rem;
+	color: #000000;
+}
+
+/* 10. This styles should apply to the paragraph in the footer */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:111]:
```suggestion
/* 10. These styles should apply to the paragraph in the footer */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	color: #ffffff;
+}
+
+/* 11. This styles should apply to all lists */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:117]:
```suggestion
/* 11. These styles should apply to all lists */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	padding: 1rem 2rem;
+	background: #caced2;
+}
+
+/* 12. This styles should apply to all list items */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:124]:
```suggestion
/* 12. These styles should apply to all list items */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	margin: 0 0 0.5rem 0;
+}
+
+/* 13. This styles should apply to all nested lists */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:130]:
```suggestion
/* 13. These styles should apply to all nested lists */
```

File: sessions/css-structure/selectors-and-cascading/css/styles.css [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+FIX_ME {
+	outline: 2px dashed red;
+	margin: 0.5rem 0 0 0;
+	background-color: #f1f1f1;
+}
+
+/* 14. This styles should apply to all list items in nested lists */
```
Comment (macfrei) [inline sessions/css-structure/selectors-and-cascading/css/styles.css:137]:
```suggestion
/* 14. These styles should apply to all list items in nested lists */
```

**Pull request merged**